graviti/openapi/user.py

```diff
@@ -21,7 +21,10 @@ def get_current_user(access_key: str, url: str) -> Dict[str, Any]:
         The response of OpenAPI.
 
     Examples:
-        >>> get_current_user("ACCESSKEY-********", "https://api.graviti.com")
+        >>> get_current_user(
+        ...     "ACCESSKEY-********",
+        ...     "https://gas.graviti.com/gatewayv2/tensorbay-open-api",
```

Comment:
may be we can keep the url in example unchanged?

Reply:
But this example is wrong, although few users will see it here.